frontend/src/components/search/SearchFilterAccordion/SearchFilterAccordion.tsx

```diff
@@ -23,40 +25,34 @@ export interface FilterOption {
   children?: FilterOption[];
 }
 
+export interface FilterOptionWithChildren {
+  id: string;
+  label: string;
+  value: string;
+  isChecked?: boolean;
+  children: FilterOption[];
+}
+
 interface SearchFilterAccordionProps {
-  initialFilterOptions: FilterOption[];
+  options: FilterOption[];
```

Comment:
@btabaska since the interface is `SearchFilterAccordionProps` it seems cleaner to me to have it as just `options` as it would follow that they are options for SearchFilterAccordion. I changed it because the "initial" didn't make sense anymore as these are not capturing state. It is great to keep in mind community devs and like `filterOptions` makes more sense for you so updating it.